apps/hash-frontend/src/shared/layout/layout-with-header/actions-dropdown.tsx

```diff
@@ -116,6 +116,9 @@ export const ActionsDropdownInner: FunctionComponent<{
               >
                 <ListItemText primary="Create Entity Type" />
               </MenuItem>,
+              <MenuItem key="org" href="/new/org" onClick={popupState.close}>
+                <ListItemText primary="Create Organization" />
+              </MenuItem>,
```

Comment:
Nit: maybe we can change this to something like this:
```Typescript
{activeWorkspace
          ? [
              { href: "/new/entity", label: "Create Entity" },
              { href: "/new/types/entity-type", label: "Create Entity Type" },
              { href: "/new/org", label: "Create Organization" },
            ].map(({ href, label }) => (
              <MenuItem key={href} href={href} onClick={popupState.close}>
                <ListItemText primary={label} />
              </MenuItem>
            ))
          : null}
```

That way we don't have to remember to add a `key` property every time we add another action item, whilst reducing the code a bit.

Reply:
Good idea – I've done this in https://github.com/hashintel/hash/pull/2788/commits/3d46ff72d911d94fad250188bf019958764afe29